packages/rx_bloc_cli/mason_templates/bricks/feature_google_login/__brick__/lib/lib_auth_google/ui_components/lib_auth_google_button.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:widget_toolkit/widget_toolkit.dart' hide ErrorModel;
+
+import '../../app_extensions.dart';
+import '../../base/common_ui_components/app_error_modal_widget.dart';
+import '../../base/extensions/async_snapshot_extensions.dart';
+import '../blocs/google_login_bloc.dart';
+
+class LibAuthGoogleButton extends StatelessWidget {
+  const LibAuthGoogleButton({
+    Key? key,
+  }) : super(key: key);
```

Comment:
Rename the file and class  to something like GoogleLoginButton (will be consistent with the service). Prefix "Lib" doesn't fit well here.